.github/workflows/vizro-qa-tests-trigger.yml

```diff
@@ -50,6 +51,8 @@ jobs:
             export INPUT_WORKFLOW_FILE_NAME=${{ secrets.VIZRO_QA_NOTEBOOKS_TESTS_WORKFLOW }}
           elif [ "${{ matrix.label }}" == "vizro-ai ui tests" ]; then
             export INPUT_WORKFLOW_FILE_NAME=${{ secrets.VIZRO_QA_VIZRO_AI_UI_TESTS_WORKFLOW }}
+          elif [ "${{ matrix.label }}" == "component library tests" ]; then
+            export INPUT_WORKFLOW_FILE_NAME=${{ secrets.VIZRO_QA_VIZRO_COMPONENT_LIBRARY_TESTS_WORKFLOW }}
```

Comment:
What is this component library tests? Are these the KPI cards used in a pure dash app?

Reply:
Yes, and now I've added link to the vizro-qa PR with the tests